### PR TITLE
Add round end feature

### PR DESCRIPTION
**Pull request by louis1706**
## Description
**Describe the changes** 

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?** (if this is a feature change)

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:

<br />

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [ ] I have checked the project can be compiled
- [ ] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [ ] I have checked no IL patching errors in the console

### Other
- [ ] Still requires more testing

### CODE REVIEW

**Review by Misaka-ZeroTwo — COMMENTED** [review]
Please use ChatGPT or Grammarly on your docs. And properly describe your changes using the template :)

File: EXILED/Exiled.API/Features/Round.cs [hunk 1]
```diff
@@ -121,6 +127,7 @@ public static int Kills
         public static int SurvivingSCPs
         {
             get => RoundSummary.SurvivingSCPs;
+            [Obsolete("This value is rewrited by NW every time it's used", true)]
```
Comment (VladTheCow) [inline EXILED/Exiled.API/Features/Round.cs:130]:
```suggestion
            [Obsolete("This value is rewritten by NW every time it's used", true)]
```

File: EXILED/Exiled.Events/EventArgs/Server/RoundEndedEventArgs.cs [hunk 2]
```diff
@@ -49,5 +49,10 @@ public RoundEndedEventArgs(LeadingTeam leadingTeam, RoundSummary.SumInfo_ClassLi
         /// Gets or sets the time to restart the next round.
         /// </summary>
         public int TimeToRestart { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether than Round Summary will be sent to all player.
```
Comment (VladTheCow) [inline EXILED/Exiled.Events/EventArgs/Server/RoundEndedEventArgs.cs:54]:
```suggestion
        /// Gets or sets a value indicating whether Round Summary will be sent to all players.
```

**Review by VladTheCow — CHANGES_REQUESTED** [review]

**Comment by louis1706** [comment]
> Please use ChatGPT or Grammarly on your docs. And properly describe your changes using the template :)

Was a fast idea i wanted to put on paper i have exam this week so i don't really have time to do more rn

**Review by VALERA771 — APPROVED** [review]

**Review by Misfiy — APPROVED** [review]

**Pull request merged**